dashboard/src/network/lib/apiArticleFunctions.js

```diff
@@ -53,3 +64,8 @@ export async function fillRequest(requestID, articleID, reportJson) {
 /// Get articles by query.
 
 /// Get article by some filter from the filterEnum.
+
+// Get recommendations
+export async function getRecommendationsForUser(userID) {
```

Comment:
Why is this function needed? Wouldn't we only want to show the currently logged in user their recs (The previous function you added achieves that) @YarivLevy81